src/main/java/org/gridsuite/network/map/model/TemporaryLimitData.java

```diff
@@ -0,0 +1,28 @@
+/**
+ * Copyright (c) 2023, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package org.gridsuite.network.map.model;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import lombok.Builder;
+import lombok.Getter;
+
+/**
+ * @author David Braquart <david.braquart at rte-france.com>
+ */
+@Builder
+@Getter
+public class TemporaryLimitData {
+    @JsonInclude(JsonInclude.Include.NON_NULL)
```

Comment:
Why name is mandatory ?
It seems everytime, it will just replicate acceptableDuration though in minutes in stead of seconds.
As a matter of symetry, order of parameters differ between network-map-server and network-modification-server.

Reply:
I dont understand the comment about acceptableDuration.

yes I re-ordered the members to be just like network-modification-server